clients/commercetools/index.js

```diff
@@ -0,0 +1,26 @@
+require('isomorphic-fetch');
```

Comment:
Didn't see this before 😅  It's because you need to import the whole library or something like that?

Reply:
Yes, it requires internally the `fetch` library globally. It is mentioned in the docs.